Fix for message about pad_token_id

ebolam opened the pull request:
Minor change to remove the warning about "Setting `pad_token_id` to `eos_token_id`:{eos_token_id} for open-end generation."

ebolam committed:
Fix for message about attention mask

ebolam committed:
update for tortoise tts

ebolam committed:
Changed tortoise tts to use environmental variables to overwrite defaults

ebolam committed:
Fix for blank actions in TTS export

ebolam committed:
Better text splitting for TTS

ebolam committed:
better text size handling for tortoise

ebolam commented:
Also added in fixes for tortoise TTS which in theory speeds up generation by x15.

Requires users running the following to install tortoise and dependencies without messing up transformers:
```
pip install git+https://github.com/neonbjb/tortoise-tts OmegaConf deepspeed
pip install torchaudio --index-url https://download.pytorch.org/whl/cu118
pip install -r requirements.txt --no-dependencies
```

Need experimental features turned on in the UI and turn on generate audio. Should background process the text into ogg files for playback in the UI (was there before, just faster now with better chunking)

henk717 commented:
We should not override the PAD id at all times just to get rid of a message, I agree with the idea of suppressing the error by replicating what HF is doing, but we should only do so if the pad token is not present.

ebolam commented:
The code should use the tokenizer's pad id always. Is there another source of pad ids?

ebolam committed:
Add story double click to load

ebolam committed:
More Audio enhancements

ebolam committed:
Merge branch 'Model_Plugins' of https://github.com/ebolam/KoboldAI into Model_Plugins

ebolam committed:
Updated audio to allow for download of single audio file with all actions in it

ebolam committed:
Modify split range for fast audio tts

ebolam committed:
Full Audio status message

ebolam committed:
Moved TTS out of experimental
Add install script for tortoise_tts and added omegaconf requirement for normal tts (was missed)
